moq-arch.md

```diff
@@ -2,87 +2,99 @@
-what CDN provides today. QuicR is pronounced something close to
-“quicker” but with more of a pirate "arrrr" at the end.
-
-The subscribe messages allow subscription to a name that includes a
-wildcard to match multiple published names, so a single subscribe can
-allow a client to receive publishes for a wide class of named objects.
-
-A typical use case is an interactive communication application,
-e.g. video conferencing, where each endpoint in the conference
-subscribes to the media from the participants in the conference and at
-the same time publishes its own media. The cloud device that receives
-the subscriptions and distributes media is called a Relay and is similar
-to an application-independent SFU in the audio/video conferencing uses
-cases and similar to a CDN cache node in traditional streaming.
-
-The Relays are arranged in a logical tree where, for a given application,
-there is an origin Relay at root of the tree that controls the
-namespace. Publish messages are sent towards the root of the tree and
-down the path of any subscribers to that named data.
-
-The QuicR protocol takes care of transmitting named objects from the
-Publisher to the Relay and from the Relay to all the subscribers of the
-named object. It provides transport services selected and tuned based on
-application requirements (with the support of underlying transport,
-where necessary) such as detecting available bandwidth, fragmentation
-and reassembly, resiliency, congestion control and prioritization of
+what CDN provides today. The subscribe messages allow subscription to a 
+name that includes a wildcard to match multiple published names, so a 
+single subscribe can allow a client to receive publishes for a wide class 
+of named objects. Objects are named such that it is unique for the
+relay/delivery network and scoped to an application. 
+
+QuicrR provides services based on application requirements 
+(with the support of underlying transport, where necessary) 
+such as detecting available bandwidth, fragmentation and 
+reassembly, resiliency, congestion control and prioritization of
 data delivery based on data lifetime and importance of data. It is
 designed to be NAT and firewall traversal friendly and can be fronted
-with load balancers. Objects are named such that it is unique for the
-relay/delivery network and scoped to an application. Subscriptions can
-include a form of wildcarding to the named object.
+with load balancers. 
+
+The Relays are arranged in a logical tree (as shown below) where, 
+for a given application, there is an origin Relay at root of the tree 
+that controls the namespace. Publish messages are sent towards the 
+root of the tree and down the path of any subscribers to that 
+named object. QuicR is designed to make it easy to implement relays 
+so that fail over could happen between relays with minimal impact to 
+the clients and relays can redirect a client to a different relay.
+
+!--
```

Comment:
We need to check but I don't think we need the !-- a start and end of figure or at least they are not coming out how we might expect

Reply:
this is fixed now